### ISSUE TITLE
JSON parse error: Cannot construct instance of `java.util.LinkedHashMap`

### ISSUE THREAD

**Issue by bhaskar-p7** [leangen/graphql-spqr-spring-boot-starter#21]
Hello, I am trying to use spring boot auto starter in my project. I am not sure what I am missing.
I am getting below error : 

> JSON parse error: Cannot construct instance of `java.util.LinkedHashMap` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value (''); nested exception is com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `java.util.LinkedHashMap` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('')\n at [Source: (PushbackInputStream); line: 1, column: 70] (through reference chain: io.leangen.graphql.spqr.spring.web.dto.GraphQLRequest[\"variables\"])",
  "path": "/graphql"

Below is my service class in spring boot application.

```java
@GraphQLApi
@Service
public class Serviceq {

	@Autowired
	PRepository pRepo;

	@Transactional
	@GraphQLQuery(name = "party")
	public Test getPByNum(@GraphQLArgument(name = "num") String num)
					throws Exception, ExecutionException {

		Future<Test> test= partyRepo.findByNum(num);
		return test.get();
	}

}
```

I am requesting with GraphiQL.

```graphql
{
  party (num: "1234"){
    id
    name
   number
  }
}
```

**Comment by bhaskar-p7** [comment]
Sorry, below is the request from graphiql.
{
party (num: "1234"){
id
name
number
}
}

**Comment by kaqqao** [comment]
Not sure where `LinkedHashMap` is coming from?
The mentioned field [`io.leangen.graphql.spqr.spring.web.dto.GraphQLRequest#variables`](https://github.com/leangen/graphql-spqr-spring-boot-starter/blob/master/graphql-spqr-spring-boot-autoconfigure/src/main/java/io/leangen/graphql/spqr/spring/web/dto/GraphQLRequest.java#L19) is of types `Map<String, Object>`, not `LinkedHashMap`.

**Comment by nngo** [comment]
I get a very similar error, trying to use 
```java
objectMapper.convertValue(map, MyClass.class);
```
Where MyClass has an `Map<String,String> field`, which I am trying to set to an empty Map via "{}"

```
java.lang.IllegalArgumentException: Cannot construct instance of `java.util.LinkedHashMap`
  (although at least one Creator exists):
  no String-argument constructor/factory method to deserialize from String value ('{}')
 at [Source: UNKNOWN; line: -1, column: -1]
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:3750) ~[jackson-databind-2.9.8.jar:2.9.8]
	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:3668) ~[jackson-databind-2.9.8.jar:2.9.8]
```

**Comment by kaqqao** [comment]
@nngo Wait... I don't see any GraphQL SPQR in your example. How does it relate? Can you post an example that reproduces the issue?

**Comment by kaqqao** [comment]
@nngo I think I found what was causing issue. It is fixed in the next release of SPQR (0.10.0), so the next release of the Starter (0.0.5) should have the fix as well.
If you keep having problems, feel free to reopen.
